docs/docs/FTSAnalyzersCharFiltersInnerMappings.md

```diff
@@ -1,56 +0,0 @@
-# FTSAnalyzersCharFiltersInnerMappings
-
-## Properties
-
-Name | Type | Description | Notes
------------- | ------------- | ------------- | -------------
-**AdditionalPropertiesField** | Pointer to **string** |  | [optional] 
```

Comment:
This is just regular clenaup. Those objects did not make any sense (they were all empty) and obfuscating our views.